sdv/constraints/tabular.py

```diff
@@ -388,93 +420,94 @@ def reverse_transform(self, table_data):
                 Transformed data.
         """
         table_data = table_data.copy()
-        diff = (np.exp(table_data[self._diff_column]).round() - 1).clip(0)
+        diff = (np.exp(table_data[self._diff_columns].values).round() - 1).clip(0)
         if self._is_datetime:
-            diff = pd.to_timedelta(diff)
+            diff = diff.astype('timedelta64[ns]')
 
-        high = self._get_high_value(table_data)
-        low = self._get_low_value(table_data)
+        high = self._get_value(table_data, 'high')
```

Comment:
It seems we are always trying to get `high` and `low`, even though we will always need only one of them, and they will need to be always accessed only once.
What about moving these further down, into the `if/else` block?

```python3
if self._drop == 'high':
    low = self._get_value(table_data, 'low')
    new_values = pd.DataFrame(diff + low, columns=self._high)
    table_data[self._high] = new_values.astype(dict(zip(self._high, self._dtype)))
...
```

Reply:
I thought about it, I feel like it is a bit redundant, I also though about:
```python3
if self._drop != 'high':
    high = self._get_value(table_data, 'high')
if self._drop != 'low':
    low = self._get_value(table_data, 'low')
```

but that also feels redundant because of the succeeding `if` statements. I'm leaning towards your suggestion or keeping it as is with `None` returns.